docs/guides/config-history.mdx

```diff
@@ -0,0 +1,13 @@
+---
+sidebar_label: Config History
+title: Config History
+---
+
+Config history for things like Feature Gates, Experiments, Dynamic Configs, and Segments can be accessed by clicking the "History" button on the top right of the page.
+![Screen Shot 2022-05-23 at 11 57 53 AM](https://user-images.githubusercontent.com/77301670/169888209-0def03c4-f079-45c7-8f6b-427ad54283e5.png)
+
+You will see a list of changes with an author and timestamp. 
+![Screen Shot 2022-05-23 at 11 58 29 AM](https://user-images.githubusercontent.com/77301670/169888219-6dd20bb4-2d22-4396-b9cb-ebe3a7307b90.png)
+
```

Comment:
Some more context to add here from confusion i've seen in slack:

This page shows each change to a given config and the time that change was published.  It is meant to make it easy to revert to a previous state, particularly if you start rolling something out that is causing issues in production.  The "preview" action will show a diff view between the current state of the gate and the previous state you selected, so you can make sure you are reverting to the correct state.

:::info Note that changes in nested configs are not listed (e.g. if this gate references another gate or segment via a "Passes Target Gate" or "User in Segment" condition, changes to the other gate or segment will not show in the history view.
:::